wordpress/user.py

```diff
@@ -0,0 +1,41 @@
+import wordpress
+from wordpress.client import APIClient
+
+
+class WPUser:
+    email = None
+    roles = None
+    user_id = None
```

Comment:
class attributes won't allow us to have multiple instances of user. They should really be instance attributes.